tina/schema.tsx

```diff
@@ -481,5 +482,38 @@ export const schema = defineSchema({
         },
       ],
     },
+    {
+      name: 'meetingLinks',
+      label: 'Meeting Links',
+      path: 'content/meeting-links',
+      format: 'json',
+      fields: [
```

Comment:
I think you can do something like this to remove the duplicated fields

```suggestion
      template: bookingTemplate 
      fields: [
```

Reply:
As per our conversation, fixed differently